camayoc/tests/qcs/ui/test_login.py

```diff
@@ -0,0 +1,37 @@
+# coding=utf-8
+"""Tests for UI login and logout.
+
+:caseautomation: automated
+:casecomponent: ui
+:caseimportance: high
+:caselevel: integration
+:requirement: Sonar
+:testtype: functional
+:upstream: yes
+"""
+from .views import LoginView, DashboardView
+
+
+def test_login_logout(browser):
+    """Login and logout using the default user.
+
+    :id: 88bbf267-d32e-44b1-934f-e69c84e5c99d
+    :description: Login and logout using the default user.
+    :steps:
+        1) Access the login page and fill the username and password fields
+        using the default user credentials.
+        2) Check if the dashboard page is displayed.
+        3) Logout and assert that the login page is shown.
+    :expectedresults: Both login and logout must work.
+    """
+    login = LoginView(browser)
+    login.username.fill('admin')
+    login.password.fill('pass')
+    login.login.click()
+
+    assert browser.selenium.title == 'Red Hat Entitlements Reporting'
+
+    dashboard = DashboardView(browser)
+    dashboard.user_dropdown.select_item('Logout')
```

Comment:
See above comment. 

I'm not certain this is better. Maybe even there should be a separate UI constants file, because there will probably be lots of text we will be working with.

Reply:
I think for now I will leave the string here. As UI test grows we decide if we would have a constants file for it or not.